dbms/src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -1431,14 +1432,58 @@ static SortDescription getSortDescription(const ASTSelectQuery & query)
 }
 
 
-void InterpreterSelectQuery::executeOrder(Pipeline & pipeline)
+void InterpreterSelectQuery::executeOrder(Pipeline & pipeline, SelectQueryInfo& query_info)
 {
     auto & query = getSelectQuery();
     SortDescription order_descr = getSortDescription(query);
     UInt64 limit = getLimitForSorting(query, context);
 
     const Settings & settings = context.getSettingsRef();
 
+    const auto& order_direction = order_descr.at(0).direction;
```

Comment:
Why only first direction is checked? Don’t understand it.

Reply:
The code is totally wrong.